Volume/Views/MainView/MagazinesList.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  MagazinesList.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 3/4/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct MagazinesList: View {
+    @State private var cancellableQuery: AnyCancellable?
+    @State private var state: MainView.TabState<Results> = .loading
+    @EnvironmentObject private var networkState: NetworkState
+
+    private func fetchContent(_ done: @escaping () -> Void = { }) {
+        // Simulate network request delay with Publication network request
+        // TODO: Replace with Magazines request
+        cancellableQuery = Network.shared.publisher(for: GetAllPublicationsQuery())
+            .map { data in data.publications.compactMap { $0 } }
+            .sink { completion in
+                networkState.handleCompletion(screen: .magazinesList, completion)
+            } receiveValue: {
+                state = .results((
+                    trendingMagazines: [Magazine](),
+                    otherMagazines: [String : [Magazine]]()
```

Comment:
Unless this take an any or something that requires explicit typing
```suggestion
                    trendingMagazines: [],
                    otherMagazines: [:]
```